schemaTypes/AddressType.js

```diff
@@ -0,0 +1,12 @@
+const { GraphQLObjectType, GraphQLString } = require('graphql');
+
+const AddressType = new GraphQLObjectType({
+  name: 'Address',
+  fields: () => ({
+    street: {
```

Comment:
You may want to add a comment saying that this is a placeholder to keep the ppl who might be using the code aware that more fields are coming